src/css/styles.css

```diff
@@ -262,7 +232,29 @@ footer {
   right: 0;
 } 
 
+.total {
+  display: flex;
+  font-size: 20px;
+  height: 25px
+}
+
 .spent-this-year {
+  margin-top: 7px;
+  margin-right: 17px;
   margin-left: 10px;
-  margin-right: 25px;
+}
+
+#total-cost-this-year {
+  margin-top: 7px;
+}
+
+.fee {
+  font-size: 11px;
+  margin-right: 17px;
+  margin-top: 5px;
+  height: 25px;
+}
+
+.hidden {
+  display: none !important;
```

Comment:
I am curious if this works without `!important`, if not I would like to look into why it is necessary for my own learning